ex_standards/ex_http_status.md

```diff
@@ -9,34 +9,34 @@ Some might ok with 2 options, choose one and justify it.
 ----
 
 * I cannot access that web if I'm not logged:
-  *  
+  *  403
 
 * I'm logged but I cannot access that page
-  * 
+  * 404
 
 * The item with that Id was deleted
-  * 
+  * 410
 
 * I started uploading, but I have to check another link to see if it is there
-  * 
+  * 428
 
 * That query parameter is wrong
-  * 
+  * 400
 
 * There are no resources with that ID
-  * 
+  * 404
 
 * My request was good, but it will take time to resolve
-  * 
+  * 408
 
 * The resource was deleted but no extra info was provided
-  * 
+  * 410
```

Comment:
este caso é bastante típico nalgúns frameworks e acostuma ser un 204 No content, que significa ok pero non che respondo nada no body. Un 200 con body vacío tamén vale.
410 é máis utilizado para HTML ou para APIs que mantañen memoría de recursos eliminados